src/configuration.rs

```diff
@@ -30,6 +31,7 @@ impl Theme {
 }
 
 #[derive(Debug, Copy, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
 pub enum Antialias {
 	#[serde(rename = "auto")]
```

Comment:
I believe you can remove the `#[serde(rename = "...")]` from this enum because now we have the `rename_all`.